.github/workflows/dracopy.yaml

```diff
@@ -0,0 +1,74 @@
+name: DracoPy
+
+on:
+  workflow_dispatch:
+    inputs:
+      dracopyVersion:
+        description: "DracoPy version"
+        required: true
+        default: "0.5.0"
+      numpyVersion:
+        description: "numpy version"
+        required: true
+        default: "2.0"
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    env:
+      CIBW_BUILD: "cp310-* cp311-* cp312-*"
+      DRACOPY_VERSION: ${{ inputs.dracopyVersion }}
+      NUMPY_VERSION: ${{ inputs.numpyVersion }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    defaults:
+      run:
+        shell: bash
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Download DracoPy
+        run: |
+          git clone https://gitlab.sauerburger.com/frank/sortednp.git \
```

Comment:
Ähh, this doesn't look right.

Reply:
Ja, ist bekannt. Tobi hat etwas vorschnell gemered, damit er Github den neuen worklfow zum testen erkennt